src/dev/mergince.md

```diff
@@ -12,6 +12,116 @@ If you need more functionality than <MainPlatformName /> CE offers, explore our
+| `COLLECT_STATISTICS`     | Boolean   | `true`    | Whether to send usage statistics for application improvements. |
+| `SERVICE_ID`             | string    |           | Deployment UUID. Auto-generated on first run. |
+​
+
+#### Security settings
+Security settings. Variables marked with asterisk* need to be modified.
+
+| Variable name            | Type      | Default   | Description |
+|--------------------------|-----------|-----------|-------------|
+| `SECRET_KEY`*            | string    |           | Secret key for authorisation, should be generated strong string. |
+| `SECURITY_PASSWORD_SALT`*| string    |           | Password salt for hashing, should be generated strong string. |
+| `BEARER_TOKEN_EXPIRATION`| integer   |  `43200`  | Lifetime of authorisation bearer token in seconds. When expired, user needs to log in again. |
+| `WTF_CSRF_ENABLED`       | Boolean   |  `true`   | Enable CSRF protection. It is strongly recommended to have this on. |
+| `WTF_CSRF_TIME_LIMIT`    | integer   |  `86400`  | Lifetime of CSRF token in seconds. When expired, user needs to refresh it. |
+​
+#### Database settings
+Mergin Maps uses PostgreSQL database to store its data. Variables marked with asterisk* need to be modified.
+​
+| Variable name             | Type      | Default   | Description |
+|---------------------------|-----------|-----------|-------------|
+| `DB_APPLICATION_NAME`     | string    | `mergin`  | Comment in database connection string to better identify connection source.  |
+| `DB_DATABASE`             | string    |`postgres` | Database to store <MainPlatformName /> tables.  |
+| `DB_HOST`                 | string    | `db`      | Database host. Mapped to docker-compose service name.  |
+| `DB_PORT`                 | integer   | `5432`    | Database port. If non-default should match port exposed in docker-compose file.  |
+| `DB_USER`*                | string    |`postgres` | PostgreSQL user to connect to <MainPlatformName /> database.  |
+| `DB_PASSWORD`*            | string    |`postgres` | PostgreSQL user password.  |
+| `DB_POOL_MAX_OVERFLOW=10` | integer   | `10`      | Database `max_overflow` limit for [SQLAlchemy](https://docs.sqlalchemy.org/en/14/core/engines.html).  |
+| `DB_POOL_SIZE`            | integer   | `2`       | Database pool size for SQLAlchemy. With overflow determines maximum of concurrent connections to database. |
+| `DB_POOL_TIMEOUT`         | integer   | `300`     | Database pool timeout for SQLAlchemy. |
+​
+#### Sending Emails
+​Mergin Maps can connect to SMTP server to send notifications. Also required for password recovery to work. Variables marked with asterisk* need to be modified.
+​
+| Variable name             | Type      | Default   | Description |
+|---------------------------|-----------|-----------|-------------|
+| `MAIL_SUPPRESS_SEND`      | Boolean   | `true`    | Whether to suppress email sending.  |
```

Comment:
let's put * here as well, as this should be changed to 0 in production deployment